diff --git a/build/builtin/main.js b/build/builtin/main.js
index f4dbc6e8325..9b7de8605f9 100644
--- a/build/builtin/main.js
+++ b/build/builtin/main.js
@@ -27,6 +27,8 @@ app.once('ready', () => {
 	window = new BrowserWindow({
 		width: 800,
 		height: 600,
+		transparent: true,
+		backgroundColor: "#00000000",
 		webPreferences: {
 			nodeIntegration: true,
 			contextIsolation: false,
diff --git a/src/vs/platform/theme/electron-main/themeMainServiceImpl.ts b/src/vs/platform/theme/electron-main/themeMainServiceImpl.ts
index 8a31973e9d1..9ac7e3e932d 100644
--- a/src/vs/platform/theme/electron-main/themeMainServiceImpl.ts
+++ b/src/vs/platform/theme/electron-main/themeMainServiceImpl.ts
@@ -297,7 +297,7 @@ export class ThemeMainService extends Disposable implements IThemeMainService {
 	private updateBackgroundColor(windowId: number, splash: IPartsSplash): void {
 		for (const window of getAllWindowsExcludingOffscreen()) {
 			if (window.id === windowId) {
-				window.setBackgroundColor(splash.colorInfo.background);
+				window.setBackgroundColor("#00000000");
 				break;
 			}
 		}
