diff --git a/src/vs/workbench/browser/parts/titlebar/titlebarPart.ts b/src/vs/workbench/browser/parts/titlebar/titlebarPart.ts
index 743f9e6ee8b..be56f0f4e1f 100644
--- a/src/vs/workbench/browser/parts/titlebar/titlebarPart.ts
+++ b/src/vs/workbench/browser/parts/titlebar/titlebarPart.ts
@@ -761,7 +761,8 @@ export class BrowserTitlebarPart extends Part implements ITitlebarPart {
 				// To benefit from LCD font rendering, we must ensure that we always set an
 				// opaque background color. As such, we compute an opaque color given we know
 				// the background color is the workbench background.
-				return color.isOpaque() ? color : color.makeOpaque(WORKBENCH_BACKGROUND(theme));
+				WORKBENCH_BACKGROUND(theme); // avoid errors
+				return color;
 			}) || '';
 			this.element.style.backgroundColor = titleBackground;
 
